examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -52,8 +62,9 @@ CHIP_ERROR CHIPCommand::Run()
     factoryInitParams.listenPort               = static_cast<uint16_t>(mDefaultStorage.GetListenPort() + CurrentCommissionerId());
     ReturnLogErrorOnFailure(DeviceControllerFactory::GetInstance().Init(factoryInitParams));
 
-    // TODO(issue #15209): Replace this trust store with file-based trust store
-    const chip::Credentials::AttestationTrustStore * trustStore = chip::Credentials::GetTestAttestationTrustStore();
+    // we assume that chip-tool is being run by default under the examples/chip-tool/out/host directory.
```

Comment:
We cannot assume this... I never run it from there. We can only use the command line path. If not provided, current directory (`.`) can be used, at best. 

We should error-out the client with an informative error message if we don't find any PAAs.